duckduckgo/macos-browser
# Fix Window menu and Dock menu

Task/Issue URL: https://app.asana.com/0/1199230911884351/1203328108135290/f
Original PR: #826

Description:

Adds support for window selector.
Access from: Menu -> Window and Dock Menu (right-click App icon in the Dock)
Each window title on the menu should be the current selected tab title.

Steps to test this PR:

Open multiple windows and on each window open a different website, settings, duckplayer, etc
- Go to Menu -> Window and check if the titles there reflect the correct selected tabs on each window
- Open Dock Menu and check if the titles there reflect the correct selected tabs on each window
- Select a window from the window menu and confirm the correct window was selected
- Change selected tabs and confirm the window title was also changed
- Navigate to another url/tab content mode and validate window title is changed
- Close windows and confirm the window title is not on the menu anymore
